### ISSUE TITLE
Internet Connection Error on Tethered Connection

### ISSUE THREAD

**Issue by jadenPete** [Thomas101/wmail#657]
* 2.3.1 Prerelease
* Debian Testing

I am getting an internet connection error while using a tethered connection. I am using EasyTether and tethering from my phone's wifi through USB. I don't really have a dedicated local ip. All of the local ip's on the network are from 192.168.1.2 through 192.168.1.255, while this one is 192.168.117.0.
![screenshot from 2017-03-13 17-49-45](https://cloud.githubusercontent.com/assets/13180372/23881111/7addddbe-0815-11e7-8207-b6c5f45019f6.png)

**Comment by Thomas101** [comment]
WMail is checking for `window.navigator.onLine` to be true for that test. If you bring up the developer tools with `Ctrl+Shift+J` and paste `window.navigator.onLine` into the console what does it come back as?

**Comment by jadenPete** [comment]
It comes back as `false`

**Comment by Thomas101** [comment]
That's odd. I would have through chromium would have handled that. If you open firefox or chrome on the machine does window.navigator.onLine come back with false too?

**Comment by jadenPete** [comment]
Nope. It returns `true` on Firefox and Google Chrome. Both browsers work perfectly.

**Comment by Thomas101** [comment]
Okay, so looks like something electron is doing. I'll try and put a test in to actually call out to the web and use that as a connection test.

**Comment by jadenPete** [comment]
Ok, thank you.
